fork.yaml

```diff
@@ -6,7 +6,7 @@ footer: |
 base:
   name: op-geth
   url: https://github.com/ethereum-optimism/op-geth
-  hash: 110c433a2469
+  hash: 641e996a2dcf1f81bac9416cb6124f86a69f1de7
```

Comment:
Looks like the OP release was done from a release branch with two additional commits
```
git log 641e996a2dcf1f81bac9416cb6124f86a69f1de7..origin/celo10-upstream
commit 110c433a24692faecce2a495c5f45efc8fcc3c33 (origin/karlb/celo10-upstream, origin/celo10-upstream, karlb/celo10-upstream, celo10-upstream)
Author: Adrian Sutton <adrian@oplabs.co>
Date:   Tue Aug 27 14:23:33 2024 +1000

    Expose method to force statedb to use singlethreaded mode even when multiple CPUs are available. (#373)

commit bb15bcd8e89ad56292805218f2bb4793470e45a5
Author: Adrian Sutton <adrian@oplabs.co>
Date:   Tue Aug 27 13:39:36 2024 +1000

    Support purely single threaded execution (#370)
    
    When only a single CPU is available:
    * Avoid creating threads in StateDB.IntermediateRoot
    * Force single threaded hashing
```

So I would say the current commit is correct (although the difference does not matter much).

Reply:
But that commit doesn't seem to exist when fordiff is run.

Here's the CI failure for the latest commit on celo10: https://github.com/celo-org/op-geth/actions/runs/12010711360/job/33478139435

Reply:
Ok, that's bad. It used to work. Maybe they didn't properly keep the branch history and it has been garbage collected. Potential solutions:
* Use a similar commit (like in this PR)
* Use the commit from this repo, since it is still preserved here. We could even point it to the celo10-upstream name to make it more readabe.